cybersixgill_actionable_alerts/manifest.json

```diff
@@ -0,0 +1,72 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b27feb80-b06f-4200-981a-e91a031d62e6",
+  "app_id": "cybersixgill-actionable-alerts",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor activity in all of our sources regarding key company assets and provide real-time alerts on incoming threats, to create dedicated playbooks",
+    "title": "cybersixgill_actionable_alerts",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
These images are showing events coming in with a Datadog logo and a source of my_apps. This will be updated to show the Cybersixgill logo and show Cybersixgill as a source once this PR has been merged and this integration has been published to your sandbox account. We strongly recommend taking new screenshots at that point with the correct source, and opening a small PR to add those new images before publishing for all customers.

Reply:
@fshaik3494 The images are still showing my_apps. Can you please update them as Ana suggested?

Reply:
@emarsha94 @anawishnoff I'm still seeing the images in my sandbox account as my_apps, how can I change this to Cybersixgill?